### ISSUE TITLE
Pressing BACKSPACE deletes mail immediately without warning

### ISSUE THREAD

**Issue by estradis** [tine20/tine20#7045]
As already described in subject, pressing the BACKSPACE key deletes selected emails without warning.

This should be stopped or at least a warning message should appear!

**Steps to reproduce:**
Select emails
Press BACKSPACE
=> Emails are gone

**Similar issues found:**
#3259 #4414 #5726

**Comment by gerryfrancis** [comment]
How do #3259 ("allow to remove messages (without moving to trash)") (feature request), #4414 ("user smtp settings"), and #5726 ("Unable to update Calendar") relate to your issue?

Please explain, thank you.

**Comment by DanielRuf** [comment]
I guess this is related to some keyup event. Can you list all steps to reproduce this (did you click into any panel?)?

**Comment by lab-at-nohl** [comment]
Hi estradis, I checked on this. In my 2019-02-07 installation [backspace] will move a mail item to Trash. Are your IMAP-folders named differently? Can you see a trashcan symbol in the folders list?

You might check on account properties - right click. See my (working) settings according to my folder names (unfortunately the window is in German)...

![grafik](https://user-images.githubusercontent.com/15079498/61477003-97476700-a98e-11e9-9ab0-5df7dc396235.png)

**Comment by estradis** [comment]
Hi dudes,

sorry for the late reply. We are all focused on a massive targeted attack on our services at moment. 

In our issue management system, the issue is described as follows:

_"The user reported that selected emails are deleted after pressing the backspace key. I tested this with my account and the mails were indeed lost. Seems to be a bug in the tine, which I have also reported in tine github. More details see there"_

I have tested this with the following results:
After pressing BACKSPACE, the message disappears from the current folder. It won't be displayed in the trash folder until the user triggers a refresh/reload event. 

I asked the responsible case worker why he linked the tasks in github as well. He told me that he searched for the keywords BACKSPACE and DEL and found these issues with similar topics, but he didn't check the content of the issues. I instructed my crew not to link cases externally in the future.

@lab-at-nohl: Screenshot is ok. We are located in southern germany.
